database/src/test/kotlin/no/nav/su/se/bakover/database/RevurderingPostgresRepoTest.kt

```diff
@@ -611,11 +646,91 @@ internal class RevurderingPostgresRepoTest {
                     attestant,
                 ),
                 skalFøreTilBrevutsending = false,
+                forhåndsvarsel = null,
                 behandlingsinformasjon = opprettet.behandlingsinformasjon,
             )
 
             repo.lagre(underkjent)
             repo.hent(underkjent.id)!! shouldBe underkjent
         }
     }
+
```

Comment:
Kunne kanskje lagt på en test på json-formatet som lagres i basen for de ulike tilfellene?